### ISSUE TITLE
Chrome46 not working properly with 8.1.7

### ISSUE THREAD

**Issue by AlfeGirl** [Popmotion/popmotion#247]
We are using version 8.1.7 on our site to make a swiper component.
When I scroll the document vertically, the swiper component scrolls horizontally.

**Comment by InventingWithMonster** [comment]
The swiper will only scroll when you start `pointer` tracking, so you'll want to detect vertical vs horizontal scrolls and handle them differently.

When you want to initiate horizontal scrolling, Chrome mobile has some annoying behaviour that means you'll want to make sure to call `e.preventDefault()` on your initial `touchstart` event, and try passing `pointer({ preventDefault: true }) to see if that works

If you have a CodePen example I can help further

**Comment by oliverzy** [comment]
Hi, I'm the author of the swiper component that AlfeGirl mentioned before. The problem is our swiper component only care horizontal scroll. If I set `pointer({ preventDefault: true }) `, it will stop browser default vertical scroll behavior.

```
      // 3. Handle Gestures
      const startX = -this.containerWidth;
      let wrapperX; // 持有translateX的值并计算速度
      let inputTracking;
      this._touchStartAction = listen(this.$refs.wrapper, 'touchstart')
        .filter(e => !this.isAnimating)
        .start(e => {
          this._clearTimeout();
          wrapperX = value({ x: startX }, wrapperStyler.set);
          inputTracking = pointer({ preventDefault: false, ...wrapperX.get() })
            .filter(v => this.canSlide(v.x - startX))
            .pipe(v => ({ x: v.x }))
            .start(wrapperX);
        });

      this._touchEndAction = listen(this.$refs.wrapper, 'touchend')
        .filter(e => !this.isAnimating)
        .start(e => {
          inputTracking.stop(); // 不再跟踪指针动作
          const endX = wrapperX.get().x;
          const deltaX = endX - startX;
          const vx = wrapperX.getVelocity().x;

          let moveNextSlide;
          if (Math.abs(vx) > 1000) {
            // Swipe Action
            this.tx = vx < 0 ? -200 : 0;
            moveNextSlide = true;
          } else {
            // Pan Action
            /* eslint-disable no-lonely-if */
            if (Math.abs(deltaX) > this.containerWidth / 2) {
              if (deltaX < 0) this.tx = -200;
              else this.tx = 0;
              moveNextSlide = true;
            } else {
              moveNextSlide = false;
            }
            /* eslint-enable no-lonely-if */
          }

          //继续向左滑动，触发父组件
          if (
            this.activePage === this.totalPages - 1 &&
            this.slideOutObj &&
            deltaX < 0 &&
            moveNextSlide
          ) {
            this._build();
            this.$emit('slideEnd');
          } else {
            this._animate(
              endX,
              this.containerWidth * (this.tx / 100),
              moveNextSlide ? this._listener : noop,
              400
            );

            this._setTimeout();
          }
        });
```

**Comment by oliverzy** [comment]
The above code is running correctly on recent mobile Chrome and Safari. It only has issue with old mobile Chrome, in this case, Chrome 46.
After looking into popmotion code, I suspect it may be related to "passive" option support, but I'm not very sure about it.

**Comment by InventingWithMonster** [comment]
I'm not certain 46 supported `passive`, but I usually engage `pointer` like this (example using Popmotion 7 but the jist of checking `angleIsVertical` still applies: https://codepen.io/popmotion/pen/dVNbmy?editors=0010

**Comment by oliverzy** [comment]
Thanks for the advice. We can try this way to see if works.

**Comment by oliverzy** [comment]
We tried to fix this issue via the methods above, but is blocked by another issue related to listen action which is used by pointer action.

Please see https://github.com/Popmotion/popmotion/pull/248 for details.

**Comment by InventingWithMonster** [comment]
Published as 8.1.9 - thanks for the PR!
